freeipa/src/test/java/com/sequenceiq/freeipa/converter/instance/template/InstanceTemplateRequestToTemplateConverterTest.java

```diff
@@ -167,7 +167,7 @@ void shouldSetEncryptionKeyAndEncryptionMethodPropertyWhenGcpEncryptionKeyPresen
         InstanceTemplateRequest source = new InstanceTemplateRequest();
         source.setInstanceType(INSTANCE_TYPE);
 
-        Template result = underTest.convert(source, CloudPlatform.GCP, ACCOUNT_ID, anyString(), "dummyEncryptionKey");
+        Template result = underTest.convert(source, CloudPlatform.GCP, ACCOUNT_ID, anyString(), "dummyEncryptionKey", null);
```

Comment:
I know it wasn't you but please don't use `anyString()` when you call the actual tested method. Please change it to some exact value.